libs/waterfall/src/lib/components/graph/layout.ts

```diff
@@ -776,17 +874,17 @@ export function fromGraph(graph: readonly Node[]) {
         node.members.forEach(member => {
           if (member.type === 'vertical') {
             rights.push(createVertical(member, undefined, node.id));
-            member.members.forEach(subMember => {
-              rights.push(createRight(subMember, undefined, member.id));
+            member.members.forEach((subMember, index) => {
+              rights.push(createRight(subMember, undefined, member.id, index));
             });
           } else {
             rights.push(createRight(member, undefined, node.id));
           }
         });
       } else if (node.type === 'vertical') {
         rights.push(createVertical(node, parentIndex[node.id]));
-        node.members.forEach(member => {
-          rights.push(createRight(member, undefined, node.id));
+        node.members.forEach((member, index) => {
+          rights.push(createRight(member, undefined, node.id, index));
```

Comment:
for better consistency.
```suggestion
        node.members.forEach((member, order) => {
          rights.push(createRight(member, undefined, node.id, order));
```